src/modules/ui.js

```diff
@@ -0,0 +1,73 @@
+const deleteActivity = (taskBlockId) => {
+  const taskBlock = document.getElementById(taskBlockId);
+  taskBlock.remove();
+  return taskBlock;
+};
+
+const createCheckBox = (checkboxState) => {
+  const checkbox = document.createElement('input');
+  checkbox.classList.add('cb');
+  checkbox.type = 'checkbox';
+  checkbox.classList.add('check_box');
+  checkbox.checked = checkboxState;
+  return checkbox;
+};
+
+const createIcon = () => {
+  const groupDiv = document.createElement('div');
+  const ellipsis = document.createElement('i');
+  ellipsis.classList.add('fa-solid', 'fa-ellipsis-vertical');
+  ellipsis.addEventListener('click', (e) => {
+    e.target.classList.add('hidden');
+    e.target.nextElementSibling.classList.remove('hidden');
+  });
+  const trash = document.createElement('i');
+  trash.classList.add('fa-solid', 'fa-trash');
+  trash.classList.add('hidden');
+
+  groupDiv.append(ellipsis);
+  groupDiv.appendChild(trash);
+
+  return groupDiv;
+};
```

Comment:
- kindly remove the functionality where one clicks on [ellipses to show the bin icon].   (https://github.com/wayungi/ToDoList-PeerReviewFeedback/blob/review_branch/src/modules/ui.js#L16)
it offers a bad user experience since the user has to do two things instead of one just to delete, and the ellipsis does not add any more functionality. You can choose to add the edit function to the ellipses or do away with one of them.